src/containers/TodoList/index.js

```diff
@@ -76,11 +76,11 @@ class TaskList extends Component {
 				<Map data={todoList.getTaskList} name="todo">
 					<List onChange={this.onChangeTask} onDelete={this.onDeleteTask} />
 				</Map>
-				<Maybe condition={todoList.getTaskList.length === 0}>
+				<Maybe condition={!todoList.getTaskList.length}>
 					<Empty />
 				</Maybe>
 
-				<Button type="danger" icon="delete" onClick={this.deleteCompletedTasks}>
+				<Button disabled={!todoList.getNumberOfCompletedTasks} type="danger" icon="delete" onClick={this.deleteCompletedTasks}>
```

Comment:
If it's possible, please make this button as a singular component and I think it's will be helpful for other starter learners.